### ISSUE TITLE
The tests for "WCAG2AAA" don't include some results from "WCAG2AA" tests

### ISSUE THREAD

**Issue by karkachev** [pa11y/pa11y#558]
I'm a developer of a web accessibility testing tool that use the pa11y-webservice app.
And durring the development I found a bug in tests for "WCAG2AAA". These results don't include some errors which return tests for "WCAG2AA".

I checked this bug in Pa11y Dashboard app and got the same issue.

I made a test for four dozen sites and see the issue is repeating on all of them.
Any of these tests doesn't include any issue of the "error" type from the "AA" level.

Here's an example made in the Pa11y Dashboard app:
[Test for "WCAG2AA"](https://pa11y.whoisaccessible.com/605241e6a8f81a66c16e93b0)
[Test for "WCAG2AAA"](https://pa11y.whoisaccessible.com/6052446ba8f81a66c16e93b2)
You can see that the "Principle1.Guideline1_4.1_4_3.G18.Fail" issue (error of AA level) from the first test, doesn't appear in the second test - https://prnt.sc/10rhagd

And the same tests on our tool:
[Test for "WCAG2AA"](https://www.accessi.org/?https://softonic.com&standard=WCAG2AA)
[Test for "WCAG2AAA"](https://www.accessi.org/?https://softonic.com&standard=WCAG2AAA)
